packages/oauth/oauth-client-browser/example/src/auth/auth-form.tsx

```diff
@@ -10,10 +10,7 @@ export function AuthForm({
   atpSignIn?: AtpSignIn
   oauthSignIn?: OAuthSignIn
 }) {
-  const defaultMethod = useCallback(
-    () => (oauthSignIn ? 'oauth' : atpSignIn ? 'atp' : undefined),
-    [],
-  )
+  const defaultMethod = oauthSignIn ? 'oauth' : atpSignIn ? 'atp' : undefined
```

Comment:
Small thing, but we could align this with the terminology we use for session managers: `oauth` and `credentials`.